solutions/usaco-787.mdx

```diff
@@ -115,7 +115,7 @@ public class rental {
                 int m = Integer.parseInt(s.nextToken());
                 int r = Integer.parseInt(s.nextToken());
 
-                // Integer instead of int is used because Collections.reverseOrder() is used later in Arrays.sort()
+                //  Integer instead of int is used because Collections.reverseOrder() is used later in Arrays.sort()
```

Comment:
uh i didn't mean like this
i meant like after a line of code, the comment needs to be two spaces after
like
```java
int x = 1;  // comment
```
instead of
```java
int x = 1; //comment
```

Reply:
aight